pkg/service/record/record.go

```diff
@@ -110,6 +126,8 @@ func (r *recorder) CaptureTraffic(path string, appCmd, appContainer, appNetwork
 	}
 
 	loadedHooks.Stop(false, stopHooksAbort)
+	tele.RecordedMock(mocksTotal)
```

Comment:
As explained this will give you wrong data.

Reply:
updated this